sqlalchemy_redshift/commands.py

```diff
@@ -425,9 +425,10 @@ def __init__(self, to, data_location, access_key_id=None,
                     '"ignore_header" parameter should be an integer'
                 )
 
```

Comment:
```python
def _upper(val):
    try:
        return val.upper()
    except TypeError:
        return None
...

format = _upper(format)
```

Reply:
TypeError will never be raised by that call; do you mean AttributeError? And if you do, that kinda defeats the point of the check. I wouldn't do that...

Reply:
Hmm. What about
```python
def _upper(v):
    return '{}'.format(v).upper()
```

Reply:
What are you trying to do? If you're trying to do sanity checking on the input type that's not the correct way to go about it, we can just cast it to str().

Reply:
I pushed an update with a str() cast.

Reply:
You cast with

```python
from typing import cast

val: Any = None
val_str: str = cast(str, val)
```

str(...) Is just a constructor

Reply:
I'm aware of the difference; I sincerely hope you're not suggesting to use this though and I don't really appreciate having my time wasted. 

Now, are you good with the current version or not?